helm/alfresco-identity-service/values.yaml

```diff
@@ -4,6 +4,8 @@ ingress:
 
 ingressHostName: "your ingress dns name"
 
+redirectUrl: "http://localhost:*"
```

Comment:
For some use cases (e.g. SAML authentication) we'll also need to allow redirect from identity service itself. It would be beneficial if we'd had an array of allowed redirects in this place.

Reply:
yes its a good point, we can have it as an array 👍